### ISSUE TITLE
Add: TVR Sport

### ISSUE THREAD

**Issue by AlbertROTV** [iptv-org/iptv#16302]
### Channel ID (required)

TVRSport.ro

### Channel Name

TVR Sport

### Stream URL (required)

https://mn-nl.mncdn.com/tvr_sport/tvr_sport/index.m3u8

### Quality

720p

### Label

None

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

**Comment by ind33deda** [comment]
Probably geo-blocked or something
![75B0AD0B-1B41-41F1-A215-BF2241B958AE](https://github.com/iptv-org/iptv/assets/164004041/36c3fa41-825d-4515-93d2-cb069332b50f)

**Comment by AlbertROTV** [comment]
The stream is for some reason not working, but it also affected all the TVR channel's streams.

**Comment by ind33deda** [comment]
Stream Link is dead, maybe i could find working links

**Comment by AlbertROTV** [comment]
https://5c3f7b934fc79.streamlock.net/live/TVRSPORT/playlist.m3u8

**Comment by AlbertROTV** [comment]
This one works.

**Comment by ind33deda** [comment]
yeah that works

**Comment by campones** [comment]
it's geoblocked

**Comment by ind33deda** [comment]
maybe i can add [Geo-blocked] in it

**Comment by ind33deda** [comment]
Can we try it with a VPN enabled?

**Comment by campones** [comment]
> Can we try it with a VPN enabled?

yes, works perfectly with a browser vpn like cyberghost extension and builtin hls video player

**Comment by ind33deda** [comment]
> > Can we try it with a VPN enabled?
> 
> yes, works perfectly with a browser vpn like cyberghost extension and builtin hls video player

send screenshot?

**Comment by campones** [comment]
do I need to give a proof?

**Comment by campones** [comment]
???

**Comment by km147** [comment]
![image](https://github.com/iptv-org/iptv/assets/14110549/8023befb-f5b4-4ee6-85db-ac3d8b135c0b)
Geoblocked

**Comment by BellezaEmporium** [comment]
Geo-blocked even with a VPN located in Bucharest. Odd.

**Comment by BellezaEmporium** [comment]
![image](https://github.com/iptv-org/iptv/assets/30985701/8f0d8a04-02fb-44d0-be5e-1fce032698b1)
If I check another channel, like TVR1, I get a "Forbidden" message.

**Comment by campones** [comment]
I don't know how you can test the website, here it doesn't work at all anyway, lots of cors error and http2 errors

**Comment by StefCoders** [comment]
All TVR channels have been blocked to their regions, so TVR Moldova would be in Moldova and TVR 1 would be Sibiu in Romania.

**Comment by campones** [comment]
dar nici cu vpn nu mai  merge? cam ciudat ca am testat acum o luna si mergea..

**Comment by StefCoders** [comment]
a blocat vpn.
